README.md

```diff
@@ -39,6 +41,16 @@ Maven users would add to their `pom.xml`
 
 A full example of a Java API client can be found [here](./examples/java/printing-client).
 
+### Python
+
+You can compile client stubs for Python with gRPC tools.
```

Comment:
I have published python stubs, can you update to something like

`pip install stellarstation`

? It will install the generated stubs and automatically install `grpcio` and `protobuf` as needed.